xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,304 @@ migrate</screen>
+	completed successfully, call the <command>crowbarctl upgrade</command>
+	command again.
+       </para>
+       <para>
+	The following commands can be helpful for analyzing issues with live migrations:
+       </para>
+       <screen>
+	nova server-migration-list
+	nova server-migration-show
+        nova instance-action-list
+	nova instance-action
+       </screen>
+       <para>
+	Note that these commands require &ostack-bare; administrator privileges.
+       </para>
+       <para>
+	The following log files may contain useful information:
+       </para>
+       <itemizedlist>
+	<listitem>
+	 <para>
+	  <filename>/var/log/nova/nova-compute</filename> on the source and
+	  target &compnode;s.
+	 </para>
+	</listitem>
+	<listitem>
+	 <para>
+	  <filename>/var/log/nova/*.log</filename> (especially log files for the
+	  conductor, scheduler and placement services) on the &contrnode;s.
+	 </para>
+	</listitem>
+       </itemizedlist>
+       <para>
+	It can happen that active instances and instances with heavy
+	loads cannot be live migrated in a reasonable time. In that case,  you
+	can abort a running live-migration operation using the <command>nova
+	live-migration-abort <replaceable>MIGRATION-ID</replaceable></command>
+	command. You can then perform the upgrade of the specific node at a
+	later time.
+       </para>
+       <para>
+	Alternatively, it is possible to force the completion of
+	the live migration by using the <command>nova
+	live-migration-force-complete
+	<replaceable>MIGRATION-ID</replaceable></command> command. However,
+	this might pause the instances for a prolonged period of time and have
+	a negative impact on the workload running inside the instance.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed during OS upgrade.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Possible reasons include an incorrect repository setup or package
+	conflicts. Check the <filename>/var/log/crowbar/node-upgrade.log</filename> log file on the
+	affected node. Check the repositories on node using the <command>zypper
+	lr</command> command. Make sure the required repositories are
+	available. To test the setup, install a package manually or run the
+	<command>zypper dup</command> command (this command is executed by the
+	upgrade script). Fix the repository setup and run the failed upgrade
+	step again. If custom package versions or version locks are in place,
+	make sure that they don't interfere with the <command>zypper dup</command> command.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node does not come up after reboot.
+       </para>
+      </question>
+      <answer>
+       <para>
+	It could happen that it takes the node too long to come up, which cause
+	a timeout. Check the node manually, make sure it is online, and repeat the step.
```

Comment:
"In some cases,  a node can take too long to reboot causing a timeout. We recommend to check the node manually..."